c2corg_ui/templates/preferences.html

```diff
@@ -0,0 +1,46 @@
+<%!
+from c2corg_common.attributes import activities
+%>
+<%inherit file="base.html"/>
+<%namespace file="helpers/common.html" import="show_title"/>
+<%block name="pagetitle"><title ng-bind="mainCtrl.page_title('Filter preferences')">${show_title('Filter preferences')}</title></%block>
+<%block name="metarobots"><meta name="robots" content="noindex,nofollow"></%block>
+
+<div class="preferences">
+  <section app-preferences class="section">
+    <h1 translate>Filter preferences</h1>
+    <p>
+      <label>
+        <input type="checkbox" ng-model="prefCtrl.followed_only">
+        <span translate>In the homepage feed see only statuses from people you follow</span>
+      </label>
+    </p>
+    <p translate>Here you may set activity and region preferences, used for instance in the activity feed.</p>
```

Comment:
I don't really like the idea of making the user interface more complicated/less clear only for a technical reason.

If later in the future we add a new feature where these preferences are used elsewhere, we can also update this translation.